vadin-demo-gui/src/main/java/de/muenchen/vaadin/ui/app/MainUI.java

```diff
@@ -271,6 +294,12 @@ private Component createNavigationMenu() {
 //            b.setIcon(testIcon.get());
             menuItemsLayout.addComponent(b);
         }
+        //creates a Help button
+        final Button helpButton = new Button("Help", event ->{ help.setVisible(!helpVisible); helpVisible = !helpVisible;});
+        helpButton.setHtmlContentAllowed(true);
+        helpButton.setPrimaryStyleName(ValoTheme.MENU_ITEM);
+        helpButton.setId("MENU_ITEM_BUTTON_HELP");
+        menuItemsLayout.addComponent(helpButton);
```

Comment:
der soll noch nach oben in die menü-bar, und wenn möglich auch mit f1 togglebar sein.